.stylelintrc.js

```diff
@@ -86,14 +108,15 @@ module.exports = {
             }
         ],
         'font-family-name-quotes': 'always-unless-keyword',
-        // Turn off rules 
+        // Turn off rules
         'selector-class-pattern': null,
         'function-no-unknown': null,
         'property-no-vendor-prefix': null,
         'value-no-vendor-prefix': null,
         'font-family-no-missing-generic-family-keyword': null,
         'scss/dollar-variable-colon-space-after': null,
         'scss/no-global-function-names': null,
-        'value-keyword-case': null
+        'value-keyword-case': null,
+        'media-query-no-invalid': null,
```

Comment:
As per https://stylelint.io/user-guide/rules/media-query-no-invalid/ this rule isn't appropriate for scss